guides/release/components/template-tag-format.md

```diff
@@ -0,0 +1,171 @@
+The template tag component format allows defining a component's template using a `<template>` tag block. This block is a first-class participant in JavaScript and TypeScript with strict mode template semantics. For this new format, the extensions `.gjs` and `.gts` are used respectively.
+
+Template tag components address a number of pain points when defining a template as a separate file, and provide a number of new capabilities:
+
```

Comment:
Most importantly: knowing where things come from and where to find them

Reply:
Adapted in rewrite.